tests/real/community.rs

```diff
@@ -729,22 +729,38 @@ async fn test_panic_in_handler() {
     let local = "test_panic_in_handler";
 
     let mut w = Worker::builder::<io::Error>()
-        .register_blocking_fn("panic", |_j| {
+        .register_blocking_fn("panic_SYNC_handler", |_j| {
             panic!("Panic inside the handler...");
         })
+        .register_fn("panic_ASYNC_handler", |j| async move {
+            // potentially going out of bounds
+            let arg1 = &j.args()[0];
+            let arg2 = &j.args()[1];
+            let _ = arg1.as_i64().unwrap() + arg2.as_i64().unwrap();
```

Comment:
Can we instead have this take a single `bool` arg that is whether or not it should call `panic!`?

Reply:
I actually _wanted_ to do it this way: we are testing here how we downcast the panic message. This is as opposed to our calling `panic!` explicitly (which we are also doing in this test)

Reply:
I think you can still test that yourself by using [`panic_any`](https://doc.rust-lang.org/std/panic/fn.panic_any.html), which allows passing in an arbitrary type to raise as the panic (thus testing downcasting).

Reply:
No that we are relaying this to `tokio::spawn`, this test restored to its previous state plus panicking inside the async handler.